webpack.config.js

```diff
@@ -152,7 +157,25 @@ const BlocksFrontendConfig = {
 			{
 				test: /\.jsx?$/,
 				exclude: /node_modules/,
-				loader: 'babel-loader?cacheDirectory',
+				use: {
+					loader: 'babel-loader?cacheDirectory',
+					options: {
+						presets: [
```

Comment:
I think this should work too and this way we don't have to define the `presets` again:
```JS
options: {
	...require( '@wordpress/babel-preset-default' ),
	plugins: [
		'@babel/plugin-transform-react-jsx',
		'@babel/plugin-transform-runtime',
		'transform-object-rest-spread',
		'transform-async-generator-functions',
	],
},
```